scipy/stats/biasedurn/fnchyppr.cpp

```diff
@@ -0,0 +1,645 @@
+/*************************** fnchyppr.cpp **********************************
+* Author:        Agner Fog
+* Date created:  2002-10-20
+* Last modified: 2014-06-14
+* Project:       stocc.zip
+* Source URL:    www.agner.org/random
+*
+* Description:
+* Calculation of univariate and multivariate Fisher's noncentral hypergeometric
+* probability distribution.
+*
+* This file contains source code for the class CFishersNCHypergeometric 
+* and CMultiFishersNCHypergeometric defined in stocc.h.
+*
+* Documentation:
+* ==============
+* The file stocc.h contains class definitions.
+* The file ran-instructions.pdf contains further documentation and 
+* instructions.
+*
+* Copyright 2002-2014 by Agner Fog. 
+* GNU General Public License http://www.gnu.org/licenses/gpl.html
+*****************************************************************************/
+
+#include <string.h>                    // memcpy function
+#include "stocc.h"                     // class definition
+
+
+/***********************************************************************
+Methods for class CFishersNCHypergeometric
+***********************************************************************/
+
+void FatalError(const char * ErrorText) {
+  throw *ErrorText;
+}
+
+CFishersNCHypergeometric::CFishersNCHypergeometric(){}
```

Comment:
I added this no-arg constructor so that I could create an instance of the class on the stack in `biasedurn.pyx`.